### DISCUSSION TITLE
Simbologías para vías

### DISCUSSION THREAD

**Discussion by ambarja** [qgispe/discusiones#14] [Ideas]
Se abre está nueva discusión a raíz de la pregunta de @JamirGarro  donde hace la respectiva pregunta.
¿Cómo podría obtener ese tipo de simbología, no me sale en mi QGIS por defecto?

<p align="center"><img src="https://user-images.githubusercontent.com/86505835/125511219-6ebd97bf-a10c-4117-8d36-398f418ee1f5.png" ></p>

**Comment by ambarja** [comment]
Una posible alternativa es la siguiente:
1. Ir a simbologías y en tipos de lineas seleccionar la categoría topo (simbología de OSM para vías)
2. Usar el generador de geometrías y crear tus propios estilos

![Captura de pantalla de 2021-07-13 15-14-55](https://user-images.githubusercontent.com/23284899/125519432-e97da92c-24cd-4ab3-9c0f-f77d5f61dd8f.png)

**Reply by JamirGarro** [reply]
si ya pude diseñar i propia simbologia, gracias :)

**Reply by ambarja** [reply]
Super, no te olvides de marcar como resuelto.
Gracias!
